src/simcams/simulated.camera.c

```diff
@@ -14,7 +14,7 @@
 
 #include "pcg_basic.h"
 
-#ifdef __AVX2__
+#if __AVX2__
```

Comment:
This is a standard macro defined by the compiler and it's proper to use ifdef here.
```suggestion
#ifdef __AVX2__
```